Remove args from LLMEdgeManager and misc cleanup

jackzhxng opened the pull request:
### Summary
Decouple `export_llama`'s `argparse.namespace` from `LLMEdgeManager`  + misc cleanup

### Test plan
CI

iseeyuan reviewed (approved):
Thanks for the refactor!

jackzhxng committed:
Remove args from LLMEdgeManager and misc cleanup (#10288)

Summary:
Decouple `export_llama`'s `argparse.namespace` from `LLMEdgeManager`  + misc cleanup


Test Plan: CI

Reviewed By: iseeyuan

Differential Revision: D73225183

Pulled By: jackzhxng
